blog/2026-08-28-dev-update-august-2026.md

```
@@ -0,0 +1,470 @@
+}
+type Child {
+  id: ID!
+  parent: Parent!
+}
+```
+
+#### Clearer Schema Validation Errors
+
+Validation errors on `schema.graphql` now give an exact location, a concise description and a suggested fix.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)